### ISSUE TITLE
Potential problem when not using a very recent version Prodigal

### ISSUE THREAD

**Issue by robinengler** [Ecogenomics/GTDBTk#94]
Hi,
Just wanted to let you know about a problem I ran into while using GTDB-Tk with Prodigal 2.6.3 (the official release, not the latest commit).

The problem is that under certain conditions (when compiled with certain versions of GCC), Prodigal will not output the correct header in .gff files as expeted by GTDB-Tk. Prodigal produced .gff files that had headers that looks like this:
```
##gff-version 3
# Sequence Data: seqnum=1;seqlen=273984;seqhdr="scaffold_0001"
# Model Data: gc_cont=38.81;transl_table=4;uses_sd=1
```

The problem is that that the line which starts with "# Model Data:" does not contain all the information it should. Specifically, GTDBTk is looking for the field "transl_table" in the "Model Data" line and it expects this field to be exactly the 5th field on this line (as in the example below), and crashes if it doesn't find it there.
Here is an example of a "# Model Data" line that follows the standard that is expected by GTDBTk:
```
# Model Data: version=Prodigal.v3.0.0-devel.1.0;run_type=Normal;model="Ab initio";gc_cont=50.79;transl_table=11;uses_sd=1
```

The latest commit of Prodigal fixes this problem, so maybe you want to updated your dependency requirements to reflect this.

Alternatively, it might also be possible to modify the way that GTDB-Tk is searching for the "transl_table" field, by searching it by name in the line rather than expecting it to be specifically at position 5. This is probably better as in this way the code becomes more resilient to changes in the .gff header format.

best,
Robin

**Comment by pchaumeil** [comment]
Hello Robin, 
Thanks for letting us know! We will update the GTDB-Tk code to take manage those 2 different headers.

Thanks,
Pierre

**Comment by pchaumeil** [comment]
Fixed in version 0.2.1
